### DISCUSSION TITLE
Fresh Contribution Model

### DISCUSSION THREAD

**Discussion by thegaryroberts** [freshframework/fresh#191] [Q&A] [answered]
Is there a contribution model? Is there a BDFL or are there a few key maintainers?

Also is there a style guide in place? e.g. for items not covered by lint: file lengths, use of lint overrides etc.

**Comment by lucacasonato** [comment, marked as answer]
> Is there a contribution model? Is there a BDFL or are there a few key maintainers?

Yes, I am the BDFL :P

More seriously, I am a core maintainer of Deno, and this is a project that we aim to move into the `denoland` organization in the not to distant future. So, it'll have the same oversight as the Deno project.

> Also is there a style guide in place? e.g. for items not covered by lint: file lengths, use of lint overrides etc.

`deno fmt` is the style guide.

**Comment by thegaryroberts** [comment]
Yeah, I know who you are dude (big fan of your work!). I just wasn't sure how hobbyist this project was going to be.

Honestly this is excellent news about plans to absorb it into the denoland organization. The main cause for lack of adoption of Deno in my peer group is the need for a robust SPA/SRR Web framework. This should change all that. Great stuff!
